Makefile

```diff
@@ -6,53 +6,46 @@ PORT ?= 3000
 
 .PHONY: dev
 dev:
-	HKNWEB_MODE='dev' pipenv run python ./manage.py runserver $(IP):$(PORT)
+	HKNWEB_MODE='dev' python ./manage.py runserver $(IP):$(PORT)
 
 .PHONY: livereload
 livereload:
-	HKNWEB_MODE='dev' pipenv run python ./manage.py livereload $(IP):$(PORT)
+	HKNWEB_MODE='dev' python ./manage.py livereload $(IP):$(PORT)
 
-setup: pipenv venv migrate
+.PHONY: setup
+setup:
+	python3 -m venv hknweb_dev
+	# Left here as a reference :))
+	# source hknweb_dev/bin/activate
 
-.PHONY: pipenv
-pipenv:
-	pip3 install --user pipenv || pip install --user pipenv
-	@echo Please add $(PIP_HOME) to your PATH variable.
-
-.PHONY: venv
-venv: Pipfile Pipfile.lock
-	pipenv install --dev
+.PHONY: install
+install:
+	python -m pip install --upgrade pip
+	python -m pip install --upgrade setuptools
+	python -m pip install -r requirements.txt
```

Comment:
+1 prefix with `<venv_dir>/bin/python` instead of just `python`.

Reply:
Also consider installing `wheel,` it may fix some issues with binary packages.

Reply:
Damn [pythonwheels](https://pythonwheels.com/) looks so cool. Not adding it in this time, adding in a comment with a TODO as a reminder.